.github/workflows/tests-smoke.yaml

```diff
@@ -68,6 +68,10 @@ jobs:
         with:
           persist-credentials: false
 
+      - name: Helm Chart Smoke Test Common label
+        run: |
+          grep -RL --exclude validate.yaml .Values.commonLabels install/kubernetes/cilium/templates/*.yaml install/kubernetes/cilium/templates/**/*.yaml 
```

Comment:
Thanks for adding this. The check should be extended a bit to provide user friendly errors in case of failure, rather than just the list of files. For instance checking the return code and echoing a bit of description alongside the list of failing files. One example could be [this one](https://github.com/cilium/cilium/blob/0b8beead6ba7a6e0c311f91baf9011355a34e183/.github/workflows/conformance-eks.yaml#L336-L340) for instance.

Reply:
I was thinking to add it make lint as well? So folks dont have to wait for CI to fail?

Reply:
Yeah, makes sense to me